tests/Microsoft.AspNet.SignalR.Tests/packages.config

```diff
@@ -1,7 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Moq" version="4.0.10827" />
+  <package id="Moq" version="4.0.10827" requireReinstallation="true" />

```

Comment:
Not sure why VS added it - maybe due to retargeting the project to net452 - not the `targetframework` is missing. I will try to update and unify versions of Moq.